IonicPortals/src/main/kotlin/io/ionic/portals/PortalFragment.kt

```diff
@@ -174,20 +189,21 @@ open class PortalFragment : Fragment {
     }
 
     private fun setupInitialContextListener() {
-        if (portal?.initialContext !== null) {
+        if (initialContext != null || portal?.initialContext !== null) {
```

Comment:
Would it be a little cleaner to just do something more like this?
```kotlin
val initialContext = initialContext ?: portal?.initialContext
if (initialContext !== null) { 
    // stuff
    val jsonOjbect = when (initialContext) {
        // more stuff
    }
}
```

Reply:
You could also chain this together:
```kotlin
val initialContext = initialContext ?: portal?.initialContext ?: return
```
If it doesn't evaluate to anything, then it just returns, otherwise, one of the initialContexts is used

Reply:
I like this